File: src/main/java/com/movisens/xs/api/exceptions/NotFoundException.java
package com.movisens.xs.api.exceptions;

import com.movisens.xs.api.models.ApiError;

public class NotFoundException extends MovisensXSException {

    private static final long serialVersionUID = 7448898720664495647L;

    public NotFoundException(ApiError apiError) {
        super(apiError);
    }
}

File: src/main/java/com/movisens/xs/api/exceptions/NotFoundException.kt
package com.movisens.xs.api.exceptions

import com.movisens.xs.api.models.ApiError

class NotFoundException(apiError: ApiError?) : MovisensXSException(apiError) {
    companion object {
        private const val serialVersionUID = 7448898720664495647L
    }
}